parser_library/test/diagnostics_check_test.cpp

```diff
@@ -236,7 +236,7 @@ TEST(diagnostics, mnemonics)
 
     ASSERT_EQ(a.parser().getNumberOfSyntaxErrors(), (size_t)0);
 
-    ASSERT_EQ(dynamic_cast<hlasm_plugin::parser_library::diagnosable*>(&a)->diags().size(), (size_t)0);
+    ASSERT_EQ(dynamic_cast<hlasm_plugin::parser_library::diagnosable*>(&a)->diags().size(), (size_t)20);
```

Comment:
```suggestion
    ASSERT_EQ(dynamic_cast<hlasm_plugin::parser_library::diagnosable*>(&a)->diags().size(), (size_t)0);
```

The same here, we should be able to create jump targets that are far enough.